<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>700</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sprites/grasses.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="healthLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sprites/VIDA_10.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="muniLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>sprites/VIDA_10_hit.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="pauseLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="heroLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>630</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="scoreLabel">
   <property name="geometry">
    <rect>
     <x>829</x>
     <y>630</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="strhpLabel">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>30</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>10/10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="munistrLabel">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>70</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>20/20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>backgroundLabel</zorder>
  <zorder>healthLabel</zorder>
  <zorder>muniLabel</zorder>
  <zorder>pauseLabel</zorder>
  <zorder>timeLabel</zorder>
  <zorder>scoreLabel</zorder>
  <zorder>heroLabel</zorder>
  <zorder>strhpLabel</zorder>
  <zorder>munistrLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
